src/Event/MissingTranslationEvent.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Yiisoft\I18n\Event;
+
+class MissingTranslationEvent
+{
+    public function __construct(string $category, string $language, string $message)
```

Comment:
Looks like constructor is not doing anything. Was that intended?

Reply:
I just kept original contructor.
 It needs to be implemented, but I didn't want to have to do this :)
May be later.